e2e/controls.py

```diff
@@ -87,6 +87,58 @@ def move_slider(self, fraction: float) -> None:
         )
         mouse.up()
 
+    def move_slider_range(self, fractionFrom: float, fractionTo: float) -> None:
```

Comment:
There's lots of duplicated logic between `move_slider` and `move_slider_range`. It would be great to reduce this duplication by introducing reusable functions. Break it down into, "what sub-problems are being solved inside these functions?" and introduce individual functions for each of those sub-problems.

I have some ideas but I would like to see what you come up with first, then we can discuss!